### PR TITLE
[Merged by Bors] - Bump operator-rs 0.41.0 and dependencies

### PR DESCRIPTION
**Pull request by maltesander**
# Description

test: https://ci.stackable.tech/view/02%20Operator%20Tests%20(custom)/job/zookeeper-operator-it-custom/51/

## Definition of Done Checklist

- Not all of these items are applicable to all PRs, the author should update this template to only leave the boxes in that are relevant
- Please make sure all these things are done and tick the boxes
 
```[tasklist]
# Author
- [ ] Changes are OpenShift compatible
- [ ] CRD changes approved
- [ ] Helm chart can be installed and deployed operator works
- [ ] Integration tests passed (for non trivial changes)
```

```[tasklist]
# Reviewer
- [ ] Code contains useful comments
- [ ] (Integration-)Test cases added
- [ ] Documentation added or updated
- [ ] Changelog updated
- [ ] Cargo.toml only contains references to git tags (not specific commits or branches)
```

```[tasklist]
# Acceptance
- [ ] Feature Tracker has been updated
- [ ] Proper release label has been added
```

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -2102,22 +2115,21 @@ checksum = "1f3ccbac311fea05f86f61904b462b55fb3df8837a366dfc601a0161d0532f20"
 
 [[package]]
 name = "tokio"
-version = "1.25.0"
+version = "1.27.0"
```
Comment (adwk67) [inline Cargo.lock:2118]: I think this means we can close this: https://github.com/stackabletech/zookeeper-operator/pull/629

**Review by adwk67 — APPROVED** [review]
LGTM % closing the renovate PR and custom tests.

**Comment by lfrancke** [comment]
Renovate PRs made irrelevant by these commits will be closed automatically closed when we run it next time

**Comment by maltesander** [comment]
bors merge

**Pull request closed without merging**